packages/web/src/components/buy-sell-modal/ConvertTab.tsx

```diff
@@ -185,7 +185,15 @@ export const ConvertTab = ({
               color='subdued'
               onClick={onChangeSwapDirection}
               aria-label='Swap token direction'
-              css={{ transform: 'rotate(90deg)' }}
+              css={{
+                transform: 'rotate(90deg)',
+                '&:hover svg': {
+                  transform: 'rotate(90deg) scale(1.1)'
+                },
+                '&:active svg': {
+                  transform: 'rotate(90deg) scale(0.98)'
+                }
+              }}
```

Comment:
messy, but works

Reply:
cool yeah this is what i landed on too but was so sad :/